### ISSUE TITLE
UI usability and speed - changes since 4.0.1

### ISSUE THREAD

**Issue by goodc0re** [spesmilo/electrum#6425]
I am missing several things in the new version. Using Electrum has just become a lot slower due to UI changes. 

Workflow before: 

Right click to select the coin or address and "Spend from"
Adjust fee
Sign and Send

Workflow since Version 4.0.1:

Right click to select the coin or address and "Spend"
Click on send tab (which no longer opens automatically)
Click on Pay (without being able to see the utxo details, nightmare)
Resize "Create transaction" window because it keeps opening too small
Adjust fee
click "Finalize"
click "Sign"
click "Broadcast" 

Also selecting Label or Address has become more time consuming now and less intuitive. 
Now it requires to steps, you have to go into a sub-menu. It was so much easier to just right-click on the label or address and copy it. 

These changes force me to downgrade and run an older version, it's affecting the time and workflow too much.

**Comment by gits7r** [comment]
Indeed, the work flow is a bit different but the transaction pop-up that appears now has a lots of finer grade tuning options, such as: rbf yes/no, set fee (choose fee determination method, select / enter proper fee rate, etc.), block height.

We are moving towards adding more features in a transaction, so that at some parts users can create their own scripts with a script maker that will support most of OP codes in the Bitcoin consensus. 

So, I think it's very good as it is now, as opposite to what we had previously and considering where we are heading. However, if you have a suggestion of a better UI can you please suggest a model? Having this one is a must, we might add another "simple" one that just pays. or I don't know. What would you like exactly? Because it's not quite possible to have the exact previous workflow, a lot of needed features have been added and this is the cleanest way.

**Comment by axelpale** [comment]
A notion from the wild: people are reverting back to 3.3.8 due these UI complexities. Simplicity is the king.

**Comment by cricktor** [comment]
There's another annoying thing on the Addresses or Coins tab. If the list of addresses with UTXOs is rather long and you need to find and select particular ones for Coin Control, it happens with varying time intervals that your selection gets erased/forgotten and the scroll position of the addresses or coins viewport gets reset to the beginning of respective list.
I observed this on previous versions of Electrum 4.1.5 and now still for 4.2.0 on Windows or Linux. As far as I can tell, it doesn't seem to matter much to which Electrum Server you're connected to. I'm running my own node with prior Electrs (latest version) and now I switched to Fulcrum Server 1.6.0 because Fulcrum is considerably faster than Electrs, especially with large address histories.

I don't know which internal process causes the reset of the addresses or coins list to the beginning of the list, but I'd love if Electrum stopped changing the list view as soon as the user starts to move around in the list. I hope you get my point.
